org_status/org_status.py

```diff
@@ -137,4 +153,8 @@ def main():
 
         org_host = Host(token, org, verbose=args.verbose)
         org_status = aggregate_org_status(org_host, threads=args.threads)
+
+        if args.export_orgs:
+            export(org_status, args.format)
```

Comment:
What if format is not passed? We should either be using sub-command which is out of scope of this PR or please add a default value to parameter definition in `get_argument_parser()`.